[Mission] S&D / TVT96 Into The Breach

HitTheMan opened the pull request:
Mission Pull Request Template

This Pull Request will:

Add the mission <mission name!> made by <author name!> to the repository.

**Mission Creation Checklist:**
- [x] Mission File
- [x] Ingame Play Area Markers
- [x] Ingame Briefings to all players
- [x] Side-based briefings
- [x] Objective Entities
- [x] Mission .conf file

**Faction(s):**
BLUFOR - Attacking
- [x] US
- [ ] RHS_US_USMC
- [ ] CDF

OPFOR - Defending
- [x] USSR
- [ ] RHS_RF_MSV

INDFOR - NA
- [ ] FIA

**Objectives/Win Conditions:**

BLUFOR:
Destroy 2x Power Generators across a river in USSR Trenches

OPFOR:
Defend the 2x Power Generators and prevent BLUFOR from crossing the river

**Terrain:**

Zarichne

**Short mission description:**

A US Platoon (+) with additional attached squads have been ordered to destroy 2x Power Generators to distrupt the USSR's communication system in the trenches outside the town of Zarichne.

While the USSR Platoon have been ordered to repel any attack conducted by the opposition, especially to maintain the communication lines with the other units in the area.

**Slotting Ratio:**

<2:1>

**Time limit:**

<60min>

**Mission Briefing Screenshot:**

![situation2](https://github.com/CoalitionArma/Coalition-Reforger-Framework/assets/108236923/a9854652-69e3-4074-8cdd-d4e7d17110e8)


**Design concept/thoughts:**

So with the addition of the new Eastern European looking map I gave it a look, I saw we had some pretty awesome trenches and more planned so I found a fun terrain feature to give the attacking force a challenge to try & cross a river that is occupied by OPFOR, with the goal to find & locate 2x Generators in the area.

With 4x map-crossings + the option to "swim" across, the players will have to think how they wish to both Defend the area as OPFOR and how to Breach the defense lines as BLUFOR, along with trench fighting in forested & housing areas or across river in support.

**Other Notes:**

- BLUFOR has a Safestart Area (BLUFOR view only)
- OPFOR has a AO Limit

- Spawn HMMWV Ambulance for BLUFOR
- Spawn S1203 Ambulance for OPFOR

- I've done my utmost to follow the standards & expectation shown to me by TanaKa on my other Mission PR along with more simplified setup thanks to the S&D mechanic existing & simplified briefings & Unit Setup

RandAlThor1 commented:
QA Feedback

- Bomb sites are too close together, bring A more south.
- 6 squads are too many for one PLT, add a coy and 2plt to BLUFOR. and split the squads between them.
- blufor squads are 10 strong, their vehicles only carry a total of 9.

- typo 
![image](https://github.com/CoalitionArma/Coalition-Reforger-Framework/assets/12282858/8152125a-fd51-42d3-8bcf-af8ed485ce57)
![image](https://github.com/CoalitionArma/Coalition-Reforger-Framework/assets/12282858/32dc5e18-c72c-4b2d-9b44-db69c5c46afb)
![image](https://github.com/CoalitionArma/Coalition-Reforger-Framework/assets/12282858/83a605b2-ebc6-4aa6-aae5-292e018657ca)

HitTheMan commented:
Changelog:

- Moved A Site further south, added trees & fences around it
- Changed BLUFOR layout;
-- Removed Medic from all Squads, making them 9
-- Added COY HQ + new Prefab Entity called Character_US_COYLead_P
-- Redone the Hierarchy + Callsigns, Company is "ALPHA"
- Fixed spellings
--
